src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -131,7 +163,7 @@ public void itemStateChanged(ItemEvent e) {
         mainPanel.add(fuzzResultTableScrollPane);
 
         add(toolbar, BorderLayout.PAGE_START);
-        add(mainPanel, BorderLayout.CENTER);
+        add(mainPanel, BorderLayout.CENTER); 

```

Comment:
Spacing and indentation changes should be submitted separately.

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines